src/main/java/domain/AbsencePenalty.java

```diff
@@ -0,0 +1,35 @@
+package domain;
+
+public enum AbsencePenalty {
+    OUT("제적", 5),
+    MEETING("면담", 3),
+    WARNING("경고", 2),
+    NORMAL("정상", 0);
+
+    public static final int LATE_TO_ABSENCE_THRESHOLD = 3;
+    private final String level;
```

Comment:
상수와 상태를 구분하기 쉽도록 절적히 줄바꿈을 통해 구분해주세요. 😃

```suggestion
    public static final int LATE_TO_ABSENCE_THRESHOLD = 3;
    
    private final String level;
```

Reply:
앗 알겠습니다!